src/ResourceManager/Websites/Commands.Websites/Cmdlets/Certificates/NewAzureWebAppSSLBinding.cs

```diff
@@ -121,11 +121,7 @@ protected override void ProcessRecord()
                     var certificateName = GenerateCertName(certificateDetails.Thumbprint, webapp.HostingEnvironmentProfile != null ? webapp.HostingEnvironmentProfile.Name : null, webapp.Location, resourceGroupName);
                     var certificate = new Certificate(
                         webapp.Location,
-#if !NETSTANDARD
```

Comment:
I don't think you can remove this

Reply:
The old block of code  pfxBlob: Convert.ToBase64String(certificateBytes) doesn't work anymore after the websites SDK update. pfxBlob was changed to a byte[] type. I think in previous versions the .NET standard build was using the byte[] type and the other build was using String type.

Reply:
👍